os-amq-launch/module.yaml

```diff
@@ -8,3 +8,8 @@ execute:
 artifacts:
 - path: ce-amq-drain-1.0.0.Final-redhat-1.jar
   md5: 65b9aba8e5a0b7d7b259a6028f85595e
+
+envs:
+    - name: "JAVA_OPTS_APPEND"
```

Comment:
I think this should be moved to the Java module, even though it's not used there directly.  Eventually it should be though, and it directly applies to Java options/JVM configuration.